cpp/examples/basic_io.cpp

```diff
@@ -27,6 +28,27 @@
 
 using namespace std;
 
+class Timer {
+ public:
+  Timer(const std::string& name) : name(name), start(std::chrono::high_resolution_clock::now()) {}
+
+  ~Timer()
+  {
+    auto end = std::chrono::high_resolution_clock::now();
+    auto start_ms =
+      std::chrono::time_point_cast<std::chrono::microseconds>(start).time_since_epoch().count();
+    auto end_ms =
+      std::chrono::time_point_cast<std::chrono::microseconds>(end).time_since_epoch().count();
+
+    std::cout << "Benchmark " << name << " takes: " << end_ms - start_ms << " microseconds"
+              << std::endl;
```

Comment:
What about a less verbose version like:
```suggestion
    std::cout << "(" << end_ms - start_ms << " us)" << std::endl;
```

and remove the `name` argument?

Reply:
Sure, done